save.go

```diff
@@ -46,6 +47,15 @@ func saveImage(image imgutil.Image, additionalNames []string, logger Logger) (Im
 	default:
 	}
 
+	manifestSize, sizeErr := image.ManifestSize()
+	if sizeErr != nil {
+		return ImageReport{}, sizeErr
```

Comment:
See [comment](https://github.com/buildpacks/lifecycle/pull/498/files?file-filters%5B%5D=.go&file-filters%5B%5D=.mod&file-filters%5B%5D=.sum#r557731001), maybe it makes sense to just return the report that we have instead of failing, so we avoid breaking builds for lower platform APIs in case we fail to get the manifest size for some reason.

Reply:
I don't think that lower platform APIs are supposed to fail in this case because the function that `ManifestSize` is using internally isn't new (see https://github.com/buildpacks/imgutil/pull/86/files).
But anyway, I'm fine that instead of failing, we can just print a message and not include the value in the report.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/fbf519cd2b6d74906571808c3207936d56eb4757